### PR TITLE
Some minor changes to the readme to fix typos

### PR DESCRIPTION
**Pull request by mikecann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,15 +162,15 @@ wolkenkit uses a number of stores to run your application. In the local developm
...
+*Please note that MongoDB must be at least version 4.2, and that you dont need to run it as a replica set (a single node cluster is fine).*
 
 For details on how to configure the databases, please have a look at the source code. This will be explained in more detail in the final version of the documentation.
 
 ### Getting help
 
 Please remember that this version is a community technology preview (CTP) of the upcoming wolkenkit 4.0. Therefore it is possible that not all provided features work as expected or that some features are missing completely.
 
-**BEWARE: Do not use the CTP for productive use, but only for getting a first impression of and evaluating the upcoming wolkenkit 4.0.**
+**BEWARE: Do not use the CTP for production use, its for getting a first impression of and evaluating the upcoming wolkenkit 4.0.**
```
Comment (goloroden) [inline README.md:173]: Thanks for the catch 👍 
However, `its` should be `it's`, shouldn't it?

File: README.md [hunk 2]
```diff
@@ -162,15 +162,15 @@ wolkenkit uses a number of stores to run your application. In the local developm
 - Redis (only for the lock store)
 - SQL Server
 
-*Please note that MongoDB must be at least version 4.2, and that you need to run it as a replica set (a single node cluster is fine).*
+*Please note that MongoDB must be at least version 4.2, and that you dont need to run it as a replica set (a single node cluster is fine).*
```
[the remark this answers is not in the document]
Comment (mikecann) [reply README.md:165]: Oh apologies, I misread it :)

File: README.md (hunk 1 above)
Comment (mikecann) [reply README.md:173]: Lol ye im not sure why im correcting grammar, its really not my strong suit

**Comment by goloroden** [comment]
@mikecann Thanks for the updated – what about

> In addition to that: Would you mind adding yourself as a contributor to the package.json file?

?

**Comment by mikecann** [comment]
@goloroden apologies, I obviously want on my PR game late last night :P

**Pull request merged**